src/components/listview/listview.js

```diff
@@ -270,52 +270,54 @@ define(['itemHelper', 'mediaInfo', 'indicators', 'connectionManager', 'layoutMan
+                    }
 
-                var imageAction = playOnImageClick ? 'resume' : action;
+                    var imageAction = playOnImageClick ? 'resume' : action;
 
-                let blurhashAttrib = '';
-                if (blurhash && blurhash.length > 0) {
-                    blurhashAttrib = 'data-blurhash="' + blurhash + '"';
-                }
+                    let blurhashAttrib = '';
+                    if (blurhash && blurhash.length > 0) {
+                        blurhashAttrib = 'data-blurhash="' + blurhash + '"';
+                    }
 
-                if (imgUrl) {
-                    html += '<div data-action="' + imageAction + '" class="' + imageClass + ' lazy" data-src="' + imgUrl + '" ' + blurhashAttrib + ' item-icon>';
-                } else {
-                    html += '<div class="' + imageClass + '">';
-                }
+                    if (imgUrl) {
+                        html += '<div data-action="' + imageAction + '" class="' + imageClass + ' lazy" data-src="' + imgUrl + '" ' + blurhashAttrib + ' item-icon>';
+                    } else {
+                        html += '<div class="' + imageClass + '">';
+                    }
 
-                var indicatorsHtml = '';
-                indicatorsHtml += indicators.getPlayedIndicatorHtml(item);
+                    var indicatorsHtml = '';
+                    indicatorsHtml += indicators.getPlayedIndicatorHtml(item);
 
-                if (indicatorsHtml) {
-                    html += '<div class="indicators listItemIndicators">' + indicatorsHtml + '</div>';
-                }
+                    if (indicatorsHtml) {
+                        html += '<div class="indicators listItemIndicators">' + indicatorsHtml + '</div>';
+                    }
 
-                if (playOnImageClick) {
-                    html += '<button is="paper-icon-button-light" class="listItemImageButton itemAction" data-action="resume"><span class="material-icons listItemImageButton-icon play_arrow"></span></button>';
-                }
+                    if (playOnImageClick) {
+                        html += '<button is="paper-icon-button-light" class="listItemImageButton itemAction" data-action="resume"><span class="material-icons listItemImageButton-icon play_arrow"></span></button>';
+                    }
 
-                var progressHtml = indicators.getProgressBarHtml(item, {
-                    containerClass: 'listItemProgressBar'
-                });
+                    var progressHtml = indicators.getProgressBarHtml(item, {
+                        containerClass: 'listItemProgressBar'
+                    });
 
-                if (progressHtml) {
-                    html += progressHtml;
+                    if (progressHtml) {
+                        html += progressHtml;
+                    }
+                    html += '</div>';
                 }
-                html += '</div>';
```

Comment:
With this, items that don't have images won't display anything. Example (from ``remotecontrol.js`` playlist):

![image](https://user-images.githubusercontent.com/10274099/85007059-48382200-b15b-11ea-9048-0d5a15fa6bf3.png)

Shall we use placeholder random backgrounds and logos? (As it's done in ``cardbuilder.js``)

Reply:
I'd remove the image for every item in that list, especially as there is already the album art near there.

But it's open for discussion, of course :)

Reply:
I think we should always show the album art, because often queues will have lots of different material. It would also look weird to have some items with an image and some without. We should really just use the placeholder image for the lists.

I **am** in agreement that the image removal is fine on the playback bar though, since it only has a single image.

Reply:
I support @dkanada here, hiding an image of _some entries_ in a list would make list look like a patchwork, let's use placeholders there. Hiding empty image in a single-item view is fine for me.